### ISSUE TITLE
Send commands to e.g. FE-C enabled ANT+ devices

### ISSUE THREAD

**Issue by JohnAZoidberg** [JohnAZoidberg/SensorWrangler#35]
Bike trainers and treadmills, implementing the FE-C ANT+ profile, can receive commands to vary the resistance and speed, respectively.

**Comment by JohnAZoidberg** [comment]
This first needs to be supported by the upstream [j-antplus](https://github.com/glever/j-antplus) library.

**Comment by TaixMiguel** [comment]
Good afternoon, did you manage to control the resistance of a bike trainer?

**Comment by JohnAZoidberg** [comment]
No, sending those commands isn't implemented in `j-antplus` yet.
And I don't have access to a bike trainer anymore.
